ios/VisionCameraCodeScanner.m

```diff
@@ -1,5 +1,19 @@
 #import <Foundation/Foundation.h>
-#import <VisionCamera/FrameProcessorPlugin.h>
 
-@interface VISION_EXPORT_SWIFT_FRAME_PROCESSOR(scanCodes, VisionCameraCodeScanner)
+#import "VisionCameraCodeScanner.h"
+#if defined __has_include && __has_include("VisionCameraCodeScanner-Swift.h")
+#import "VisionCameraCodeScanner-Swift.h"
+#else
+#import <VisionCameraCodeScanner/VisionCameraCodeScanner-Swift.h>
```

Comment:
where's this -Swift.h file located?

Reply:
It's an auto-generated file. The location is slightly different depending on whether the project is built with `use_frameworks! :linkage => :static`.

<img width="770" alt="image" src="https://github.com/rodgomesc/vision-camera-code-scanner/assets/129107344/14271802-756c-4ac7-8679-e00567c17785">

Reply:
another thing, i did install this version. but whenever open the camera with frameProcessor, it crashes. any idea?

Reply:
Can you provide any more details of the crash (e.g., logs, exception messages) or your test environment?

Reply:
I experienced the same issue. Camera works great but when i attach `frameProcessor` it crashes the app. This is the log i get from the xCode. Do you know what could be a solution for this issue?

```
VisionCamera.didSetProps(_:): Updating 16 prop(s)...
VisionCamera.configureCaptureSession(): Configuring Session...
VisionCamera.configureCaptureSession(): Initializing Camera with device com.apple.avfoundation.avcapturedevice.built-in_video:6...
VisionCamera.configureCaptureSession(): Adding Video input...
VisionCamera.configureCaptureSession(): Adding Video Data output...
VisionCamera.invokeOnInitialized(): Camera initialized!
VisionCamera.configureCaptureSession(): Session successfully configured!
VisionCamera.configureFormat(): Configuring Format...
VisionCamera.configureFormat(): Format successfully configured!
VisionCamera.configureDevice(): Configuring Device...
VisionCamera.configureDevice(): Device successfully configured!
VisionCamera.didSetProps(_:): Starting Session...
VisionCamera.didSetProps(_:): Started Session!
libc++abi: terminating due to uncaught exception of type facebook::jsi::JSINativeException: Compiling JS failed: 1:1:invalid empty parentheses '( )' Buffer size 3 starts with: 280a29
```
Btw thanks @rkmackinnon for the effort!